oak-doc/src/site/markdown/query/indexing.md

```diff
@@ -426,6 +426,21 @@ NRT indexing expose a few configuration options as part of the [LuceneIndexProvi
   to hold Lucene documents for indexing in the `nrt` mode. 
   The default size is 10000.
 
+## <a name="superseding"></a> Superseding an index
```

Comment:
This page currently uses https://en.wikipedia.org/wiki/Capitalization#Title_case so in theory you should use "Superseding an Index". I would do that for consistency. However, I know for most people this isn't important. It's up to you.

Reply:
No issues. I have changed it.